src/main/java/com/fibermc/essentialcommands/types/NamedLocationStorage.java

```diff
@@ -46,17 +39,12 @@ public void loadNbt(NbtElement nbt) {
         }
     }
 
-    public MinecraftLocation putCommand(String name, MinecraftLocation location) throws CommandSyntaxException {
+    public MinecraftLocation putCommand(String name, MinecraftLocation location) {
         return putCommand(name, new NamedMinecraftLocation(location, name));
     }
 
-    private MinecraftLocation putCommand(String name, NamedMinecraftLocation location) throws CommandSyntaxException {
-        if (this.get(name) == null) {
-            return super.put(name, location);
-        } else {
-            throw CommandUtil.createSimpleException(
-                ECText.getInstance().getText("cmd.home.set.error.exists", TextFormatType.Error, Text.literal(name)));
-        }
+    private MinecraftLocation putCommand(String name, NamedMinecraftLocation location) {
```

Comment:
I think I'd like to continue to have this prevent overwrites. In a way, it makes it harder to inadvertently add a feature that doesn't force confirmation to overwrites a home (e.g. I'd expect to, in code, have to explicitly remove the existing home, then add the new one)

Reply:
reverted the changes to the file